src/VirtualClient/VirtualClient.Dependencies/RedisPackageInstallation.cs

```diff
@@ -154,5 +160,35 @@ await this.systemManager.PackageManager.RegisterPackageAsync(redisPackage, cance
                             .ConfigureAwait(false);
 
         }
+
+        private async Task InstallOnAzLinuxAsync(EventContext telemetryContext, CancellationToken cancellationToken)
```

Comment:
have you tried tdnf?

Reply:
tdnf also works, is it preferred!?

Reply:
on AzLinux yes, on RHEL still use dnf. You don;t have to change it right now, you could add a comment that tdnf also works. I heard AzLinux eventually wants to move to tdnf entirely